app/assets/javascripts/jschannel.js

```diff
@@ -0,0 +1,620 @@
+/*
```

Comment:
Actually the jschannel library is used in two different context. It's first injected in the page where the widget is embedded and I use this file to only include what's necessary but it is also used in the widget iframe itself and in this case it is coming from the js bundle  packaged by the rails asset pipeline. Few things I'd like to change, 1) bundle and minify all the js injected in the host page 2) only use the necessary js and css assets for the widget itself excluding everything that is/will be used in the web site.